mobile/ios/ArcadiaAssistant.xcodeproj/project.pbxproj

```diff
@@ -137,6 +137,20 @@
 			remoteGlobalIDString = 83CBBA2E1A601D0E00E9B192;
 			remoteInfo = React;
 		};
+		1E121D0A1FF2AA9800E8DD05 /* PBXContainerItemProxy */ = {
```

Comment:
what's this for?

Reply:
As I said these changes were done by Xcode automatically - it makes some changes according to some Apple's rules sets (not documented), so it's not manually/intentionally change at all.